schema/home/results.js

```diff
@@ -18,8 +18,8 @@ const RESULTS_SIZE = 20;
 
 const moduleResults = {
   active_bids: () => [],
-  iconic_artists: () => {
-    return iconicArtists().then((artists) => {
+  trending_artists: () => {
```

Comment:
lol, fair fair.

I just talked with @broskoski and what we’re going to do is revert this so the name remains the same and the implementation can be changed in the future to the definition of artists with most inquiries on their works.